src/api/app/models/project.rb

```diff
@@ -1417,6 +1417,15 @@ def reopen_release_targets
     Backend::Connection.post "/build/#{URI.escape(name)}?cmd=wipe"
   end
 
+  # lock the project for the scheduler for atomic change when using multiple operations
+  def suspend_scheduler
+    Backend::Connection.post "/build/#{URI.escape(name)}?cmd=suspendproject"
```

Comment:
Can you make use of the Backend::Api calls? I mean, add those methods there and use the ConnectionHelper methods that we are using there too.

Reply:
that way?

Reply:
Yes! Thanks 💋